src/main/java/com/google/cloud/resourcemanager/ResourceManager.java

```diff
@@ -337,4 +337,14 @@ public static ProjectListOption fields(ProjectField... fields) {
    *     Platform Services</a>
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
+
+  /**
+   * Returns permissions that the caller has on the specified Organization.
+   *
+   * @throws ResourceManagerException upon failure
+   * @see <a href=
+   *     "https://cloud.google.com/resource-manager/reference/rest/v1/organizations/testIamPermissions">
+   *     Resource Manager testIamPermissions</a>
+   */
+  List<Boolean> testOrgPermissions(String resource, List<String> permissions);
 }
```

Comment:
The doc is not clear here: "Returns permissions that the caller has on the specified Organization."
1) what is 'the specified Organization'?  do you mean resource?
2) 'Returns permissions', but method does not return permissions, it returns list of boolean.
3) What is expected in the 'List<String> permissions' parameter? 

I think you need to add
@param resource The resource for which the policy detail is being requested.
@param permissions the set of permissions to check for the resource. Permissions with wildcards (such as '*' or 'storage.*') are not allowed

IMHO, returning List<String> would be much useful here.

Reply:
Thanks for this but am disagree with your opinion as method itself said that it's only used to test the permission.means it's only check whether this permissions is exists on resource or not.

Reply:
I doubt that such API is very convenient, but if there are similar method around, that's fine.
Anyway, the doc should be precise. If you believe, that returning list of boolean is what is needed, please update the doc.

Reply:
done java doc updated with proper description.